python/Ganga/GPIDev/Lib/File/OutputFileManager.py

```diff
@@ -209,38 +194,18 @@ def getWNCodeForDownloadingInputFiles(job, indent):
     Generate the code to be run on the WN to download input files
     """
 
+    if job.master is not None:
+        job = job.master
```

Comment:
Just replaced this with better logic to deal with more complex splitters which this overlooks. Incoming commit to fix this.